crates/fluvio-cli/src/client/topic/describe.rs

```diff
@@ -37,9 +42,33 @@ impl DescribeTopicsOpt {
         debug!("describe topic: {}, {:?}", topic, output_type);
 
         let admin = fluvio.admin().await;
-        let topics = admin.list::<TopicSpec, _>(vec![topic]).await?;
 
-        display::describe_topics(topics, output_type, out).await?;
+        let topics_with_system = admin
+            .list_with_config::<TopicSpec, String>(
+                ListRequest::new(ListFilters::from(topic.as_str()), true).system(self.system),
+            )
+            .await?;
+
+        let partitions = admin
+            .list_with_config::<PartitionSpec, String>(
+                ListRequest::new(ListFilters::from(topic.as_str()), true).system(self.system),
+            )
+            .await?;
+
+        let filtered_partitions = partitions
```

Comment:
let's move this to separate function and add unit test

Reply:
Done